### PR TITLE
Update transfer-tokens.md

### PR DESCRIPTION
**Pull request by eminmtas**
## Description
Additional information was needed for users to transfer funds in Scroll Sepolia Testnet. I added some additional information to the `transfer-tokens.md` file.

## Changes
- updated `transfer-tokens.md` file

### CODE REVIEW

File: src/content/docs/en/user-guide/transfer-tokens.md [hunk 1]
```diff
@@ -1,18 +1,21 @@
 ---
 section: gettingStarted
-date: Last Modified
+date: 01.27.2024
```
Comment (dghelm) [inline src/content/docs/en/user-guide/transfer-tokens.md]: We haven't had any sort of standard around using the date field here afaik, so let me check on this before approving.

**Review by dghelm — COMMENTED** [review]
Thanks for the PR! Looks good, but may request changes on the date field -- will get back to you soon.

**Comment by eminmtas** [comment]
Okay ser, waiting for it

**Comment by eminmtas** [comment]
just changed the date section to the previous one. it should be good to merge now. thanks

**Comment by dghelm** [comment]
Thanks for the contribution, but after consideration we've decided to keep the current description, as this "step" in getting started comes after the wallet setup and faucet pages, and users should be setup by this point in the walkthrough.

**Pull request closed without merging**